### PR TITLE
[TASK] Update frontend editing to version ''1.2.3''

### PR DESCRIPTION
**Pull request by MattiasNilsson**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -44,6 +44,6 @@
     "typo3-ter/pxa-form-enhancement": "~3.0.0",
     "typo3-ter/pxa-cookie-bar": "~1.0.7",
     "svewap/realurl-404-multilingual": "~1.0.9",
-    "typo3-ter/frontend-editing": "~1.2.2"
+    "typo3-ter/frontend-editing": "~1.2.3"
```
Comment (websi) [inline composer.json:47]: Why is this needed? The given version contraint already include version "1.2.3".
Or there is a reason, why the extension "theme_t3kit" not working with version 1.2.2 of "frontend_editing"?

See https://getcomposer.org/doc/articles/versions.md

Comment (MattiasNilsson) [reply composer.json:47]: @websi I just want to make sure that we always will run the version 1.2.3 as it contains some nifty improvements.

Comment (websi) [reply composer.json:47]: This is ensured for both variants.
You should use `composer update t3kit/theme-t3kit --with-dependencies`
